packages/flight-icons/svg-original/twilio-color-24.svg

```diff
@@ -0,0 +1,5 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+<g id="size=24">
+<path id="Vector" d="M12 1C18.072 1 23 5.928 23 12C23 18.072 18.072 23 12 23C5.928 23 1 18.072 1 12C1 5.928 5.928 1 12 1ZM12 3.904C7.512 3.904 3.904 7.512 3.904 12C3.904 16.488 7.512 20.096 12 20.096C16.488 20.096 20.096 16.488 20.096 12C20.096 7.512 16.488 3.904 12 3.904ZM14.728 12.44C15.9916 12.44 17.016 13.4644 17.016 14.728C17.016 15.9916 15.9916 17.016 14.728 17.016C13.4644 17.016 12.44 15.9916 12.44 14.728C12.44 13.4644 13.4644 12.44 14.728 12.44ZM9.272 12.44C10.5356 12.44 11.56 13.4644 11.56 14.728C11.56 15.9916 10.5356 17.016 9.272 17.016C8.00837 17.016 6.984 15.9916 6.984 14.728C6.984 13.4644 8.00837 12.44 9.272 12.44ZM14.728 6.984C15.9916 6.984 17.016 8.00837 17.016 9.272C17.016 10.5356 15.9916 11.56 14.728 11.56C13.4644 11.56 12.44 10.5356 12.44 9.272C12.44 8.00837 13.4644 6.984 14.728 6.984ZM9.272 6.984C10.5356 6.984 11.56 8.00837 11.56 9.272C11.56 10.5356 10.5356 11.56 9.272 11.56C8.00837 11.56 6.984 10.5356 6.984 9.272C6.984 8.00837 8.00837 6.984 9.272 6.984Z" fill="#F12E45"/>
```

Comment:
On the Twilio website the red of the logo is `#F22F46` while here is `#F12E45`. Is it expected?

Reply:
Eagle eyes! lol

Reply:
Hmm, that's the color provided from the icon download. I can update it to what they have on their website!